valinta-tulos-service/src/main/scala/fi/vm/sade/valintatulosservice/valinnantulosServlet.scala

```diff
@@ -109,14 +109,13 @@ class ValinnantulosServlet(valinnantulosService: ValinnantulosService,
   get("/hakemus/", operation(valinnantuloksetHakemukselleSwagger)) {
     contentType = formats("json")
     implicit val authenticated = authenticate
-    authorize(
-      Role.SIJOITTELU_READ,
-      Role.SIJOITTELU_READ_UPDATE,
-      Role.SIJOITTELU_CRUD,
-      Role.ATARU_KEVYT_VALINTA_READ,
-      Role.ATARU_KEVYT_VALINTA_CRUD
-    )
-    Ok(parseHakemusOid.right.map(valinnantulosService.getValinnantuloksetForHakemus(_, auditInfo)))
+    val hakemusOid = parseHakemusOid.fold(throw _, x => x)
```

Comment:
`x => x`:n tilalla voi vissiin käyttää myös `Predef`:in `identity`ä, mutta tuo `x => x` on kyllä aika lyhyt ja selkeä myös.